package.json

```diff
@@ -184,7 +184,7 @@
     "babel-plugin-transform-react-remove-prop-types": "0.4.21",
     "chalk": "2.4.1",
     "concurrently": "4.1.0",
-    "connected-react-router": "5.0.1",
+    "connected-react-router": "6.0.0-beta.1",
```

Comment:
Maybe we should hold the PR until the stable release of `connected-react-router`.

Reply:
Agree.

Reply:
stable version released